Makefile

```diff
@@ -153,12 +99,128 @@ e2e:
 
 	kubectl delete --filename config/samples
 
-# ==================================
-# Download: tools in ${BIN}
-# ==================================
-${BIN}:
-	mkdir -p ${BIN}
+##@ Build
+
+.PHONY: build
+build: manifests generate fmt vet ## Build manager binary
+	go build -o bin/manager main.go
+
+.PHONY: run
+run: manifests generate fmt vet ## Run a controller from your host.
+	go run ./main.go
+
+# If you wish built the manager image targeting other platforms you can use the --platform flag.
+# (i.e. docker build --platform linux/arm64 ). However, you must enable docker buildKit for it.
+# More info: https://docs.docker.com/develop/develop-images/build_enhancements/
+.PHONY: docker-build
+docker-build:  ## Build docker image with the manager.
+	docker build \
+		--build-arg VERSION=$(VERSION) \
+		--tag ${IMG} \
+		--file Dockerfile \
+		${CURDIR}
```

Comment:
I need to additionally pass `--load` flag to this to load it into local Docker registry if I want to load it into a kind cluster next as per the Readme. Does it work for you without?

Reply:
It works in the E2E tests too.